web_external/templates/body/legal/privacy_policy.html

```diff
@@ -0,0 +1,52 @@
+<p style="text-align:center;"><strong>International Society For Digital Imaging Of The Skin (&quot;ISDIS&quot;)</strong></p>
+<p style="text-align:center;"><strong>International Skin Imaging Collaboration Project (“ISIC”)</strong></p>
+<p style="text-align:center;"><strong>Privacy Policy</strong></p>
+<p><strong>Last Revised: October 3, 2016</strong></p>
+<p>We, Memorial Sloan Kettering Cancer Center (“<strong><em>MSKCC</em></strong>”), provide this Privacy Policy to inform users of our policies and procedures regarding the collection, use, and disclosure of personal information received from users of this Site, located at http://isdis.net/isic-project/public-image-archive-of-skin-lesions/ (the <strong>&quot;<em>Site</em></strong>&quot;). References to MSKCC include &quot;<strong><em>MSKCC Protected Parties</em></strong>,&quot; which means MSKCC and its subsidiaries, affiliates, partners, directors, officers, employees, agents, contractors, successors, licensees, or assignees (both jointly and severally).</p>
```

Comment:
@msmolens Ok, that's a safe bet. I think we should get this live and solicit feedback first. Once the stakeholders are satisfied with things, we can revisit cleaning up the formatting.